projects/driver_controls/rules.mk

```diff
@@ -17,3 +17,5 @@ $(T)_EXCLUDE_TESTS := gpio_expander magnetic_sensor throttle_calibration control
 endif
```

Comment:
You may want to add your mech calibration to this to prevent it from being built on x86